guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -34,16 +45,40 @@ Alternatively, to prevent downtime caused by stopping the service, you can use t
 [options="nowrap" subs="+quotes,attributes"]
 ----
 ifdef::satellite[]
-# subscription-manager repos --enable {RepoRHEL7ServerSatelliteToolsProductVersion}
+# subscription-manager repos --enable {RepoRHEL8ServerSatelliteUtils}
 endif::[]
 # {foreman-maintain} packages unlock
-# yum install install python-pulp-manifest -y
+# {package-install} python-pulp3-manifest
 # {foreman-maintain} packages lock
 ifdef::satellite[]
-# subscription-manager repos --disable {RepoRHEL7ServerSatelliteToolsProductVersion}
+# subscription-manager repos --disable {RepoRHEL8ServerSatelliteUtils}
 endif::[]
 ----
 This installs the package without downtime.
+** For RHEL 7:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install-project} tfm-pulpcore-python3-pulp_manifest
+----
++
+Note that this command stops the {Project} service and re-runs {foreman-installer}.
+Alternatively, to prevent downtime caused by stopping the service, you can use the following:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+ifdef::satellite[]
+# subscription-manager repos --enable {RepoRHEL8ServerSatelliteUtils}
```

Comment:
Previously the repositories were already enabled. Any reason they're enabled here again (other than that is was done previously for EL7)?

I'd suggest to drop the enable/disable inline here.

Reply:
dropped.